libnmstate/state.py

```diff
@@ -30,6 +30,43 @@
 from libnmstate.prettystate import format_desired_current_state_diff
 from libnmstate.schema import Interface
 from libnmstate.schema import InterfaceType
+from libnmstate.schema import Route
+
+
+class _Route(object):
+    def __init__(self, route):
+        self.state = copy.deepcopy(route)
+        if route.get(Route.STATE) != Route.ABSENT:
+            if Route.TABLE_ID not in route:
```

Comment:
Please call this section by a proper name under its own function.
I guess it is something like: `complement_defaults`.

I suggest you also mention in the commit message the `why` part. It is unclear why you want to do it.
I would even argue that this does not need to be included here if the explanation for it is seen in a later change.

Reply:
Why do I need a function if that function is only used for one time and it only contains 6 lines?
How hard is that for a developer to understand I am filling the default value if missing.

Reply:
Functions are also used to structure, abstract and describe what is performed (vs how). Based on your reasoning we will end up with a 1000 lines in a function.
The reader should read the code like a story and decide if he wants details or not (what vs how).